specification/storage/resource-manager/Microsoft.Storage/stable/2021-09-01/common.json

```diff
@@ -82,9 +82,9 @@
         },
         "days": {
           "type": "integer",
-          "minimum": 1,
+          "minimum": 0,
```

Comment:
Per my testing, it looks when no enabled blob DeleteRetentionPolicy, the "days" will not be return , instead of return "0".
And there's another property "enabled" to indicate if DeleteRetentionPolicy is enabled or not. 

So the change looks not aligned with server behavior, and not necessary.

Then why we add this change?